x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/embeddings/CohereEmbeddingsServiceSettings.java

```diff
@@ -22,31 +22,30 @@
 
 import java.io.IOException;
 import java.util.EnumSet;
-import java.util.Locale;
 import java.util.Map;
 import java.util.Objects;
 
 import static org.elasticsearch.xpack.inference.services.ServiceUtils.extractOptionalEnum;
-import static org.elasticsearch.xpack.inference.services.ServiceUtils.extractRequiredString;
 
 public class CohereEmbeddingsServiceSettings implements ServiceSettings {
     public static final String NAME = "cohere_embeddings_service_settings";
 
     static final String EMBEDDING_TYPE = "embedding_type";
-    static final String EMBEDDING_TYPE_BYTE = "byte";
 
     public static CohereEmbeddingsServiceSettings fromMap(Map<String, Object> map, ConfigurationParseContext context) {
         ValidationException validationException = new ValidationException();
         var commonServiceSettings = CohereServiceSettings.fromMap(map, context);
-        translateEmbeddingType(map, context);
-
-        CohereEmbeddingType embeddingTypes = extractOptionalEnum(
-            map,
-            EMBEDDING_TYPE,
-            ModelConfigurations.SERVICE_SETTINGS,
-            CohereEmbeddingType::fromString,
-            EnumSet.allOf(CohereEmbeddingType.class),
-            validationException
+
+        CohereEmbeddingType embeddingTypes = Objects.requireNonNullElse(
```

Comment:
Should we looking for a new field called `element_type` to be inline with elasticsearch's field name?